runtime/permissions/lib.rs

```diff
@@ -1641,6 +1642,91 @@ impl PermissionsContainer {
     self.0.lock().env.check_all()
   }
 
+  #[inline(always)]
+  pub fn check_sys_all(&mut self) -> Result<(), AnyError> {
+    self.0.lock().sys.check_all()
+  }
+
+  #[inline(always)]
+  pub fn check_ffi_all(&mut self) -> Result<(), AnyError> {
+    self.0.lock().ffi.check_all()
+  }
+
+  /// This checks to see if the allow-all flag was passed, not whether all
+  /// permissions are enabled!
+  #[inline(always)]
+  pub fn check_was_allow_all_flag_passed(&mut self) -> Result<(), AnyError> {
+    self.0.lock().all.check()
+  }
```

Comment:
Might be good to have a unit test for this one

Reply:
Adding this to a follow-up.